engine/crates/engine-v2/schema/tests/conversion.rs

```diff
@@ -95,7 +95,17 @@ type User
 
 #[test]
 fn should_not_fail() {
-    let FederatedGraph::V1(graph) = FederatedGraph::from_sdl(SCHEMA).unwrap();
-    let config = config::VersionedConfig::V1(graph).into_latest();
+    let graph = FederatedGraph::from_sdl(SCHEMA).unwrap().into_latest();
```

Comment:
Will add a function `config::latest::Config::from_schema()` if you don't^^

Reply:
Yes it was already tedious here, can do in this branch.

Reply:
done